src/plotApplications/requests.js

```diff
@@ -75,6 +75,41 @@ export const deleteUploadRequest = (
   );
 };
 
-export const editInfoCheckItemRequest = (payload: Object): Generator<any, any, any> => {
+export const editApplicantInfoCheckItemRequest = (payload: Object): Generator<any, any, any> => {
   return callApi(new Request(createUrl(`information_check/${payload.id}/`), {method: 'PATCH', body: JSON.stringify(payload)}));
 };
+
+export const editTargetInfoCheckItemRequest = (payload: Object): Generator<any, any, any> => {
+  return callApi(new Request(createUrl(`target_status/${payload.id}/`), {method: 'PATCH', body: JSON.stringify(payload)}));
```

Comment:
I don't know if this a bug on my env or something, but currently the payload.id is undefined and it tries to call api with that resulting an error.

Reply:
Hmm... Can you check if the target statuses exist in the answer in the first place when fetched from the API? As it is right now, the form always shows up even if it's missing, as in the future there always should be a matching target status object for each target. If there isn't, then that application is in a broken state for one reason or another. Otherwise, there might be something I need to fix (though: works for me at the moment).

Reply:
It was faulty form. the target was not dublicable.